oidc-controller/api/core/oidc/issue_token_service.py

```diff
@@ -22,6 +22,18 @@ class Claim(BaseModel):
     value: str
 
 
+# Used as a work around since these are represented as dictionaries in this case
+class ReqAttrDict(TypedDict, total=False):
+    names: List[str]
+    label: Optional[str]
+    restrictions: List[AttributeFilter]
+
+
+class RevealedAttributeDict(TypedDict, total=False):
+    sub_proof_index: int
+    values: dict
```

Comment:
Wondering if these belong to a `models` source file, rather than in here. Thoughts? Additionally, since this is extremely similar to the model in `api/verificationConfigs/models.py` would there be a way of making things work without two type definitions (that then need to be maintained)?

Reply:
I was able to adapt the code to use the existing version of ReqAttr and migrated RevealedAttribute to simply be a TypedDict since it is only used here and if we decided to include it in a Pydantic class this is supported. Pydantic just doesn't support inheriting from both BaseModel and TypedDict since BaseModel it's self does not inherit from TypedDict.